CleverPush/Source/CPStoriesController.m

```diff
@@ -329,6 +340,13 @@ - (void)userContentController:(WKUserContentController *)userContentController d
         if (![currentIndex isEqualToString:scriptMessageIndex]) {
             return;
         }
+
+        if (self.readStories.count > 0) {
```

Comment:
Please fix the tracking:

"Opened" should be tracked when clicking on a bubble item.

"Shown" should be tracked when the bubbles are rendered.